pkg/scheduler/core/scheduling_queue.go

```diff
@@ -180,6 +186,8 @@ func NominatedNodeName(pod *v1.Pod) string {
 // pods that are already tried and are determined to be unschedulable. The latter
 // is called unschedulableQ.
 type PriorityQueue struct {
+	stop  <-chan struct{}
+	clock util.Clock
 	lock sync.RWMutex
```

Comment:
You need to add a space behind `sync.` in these two lines to make gofmt happy.

Reply:
Done, thank you.